lib/usb_connection.js

```diff
@@ -228,10 +228,13 @@ util.inherits(USB.Scanner, EventEmitter);
 
 USB.Scanner.prototype.start = function() {
   var self = this;
+  if (haveusb) {
```

Comment:
@johnnyman727 It causes from my work on reference machines. Is this behavior fixed by a newer part of code ? I belief there are many professional companies provide VMs for new employees. And if they did things wrong, they need that warning.